azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/OSProfile.java

```diff
@@ -91,15 +90,52 @@ public static LinuxConfiguration create(final String disablePasswordAuthenticati
 
       @AutoValue
       public abstract static class WinRM {
+          public enum Protocol {
+
+              HTTP("http"),
+              HTTPS("https"),
+              UNRECOGNIZED("Unrecognized");
```

Comment:
@bostko I think this is just a jclouds convention, of including "UNRECOGNIZED" for if we come across an unknown value (rather than it setting the field to null or throwing an exception).